sslbl/package.yaml

```diff
@@ -0,0 +1,61 @@
+id: sslbl
+name: SSLBL
+author: Tenzir
+author_icon: https://github.com/tenzir.png
+package_icon: |
+  https://raw.githubusercontent.com/tenzir/library/main/sslbl/package.svg
+description: |
+  The [SSLBL](https://sslbl.abuse.ch/) package makes available a lookup table
+  with SHA1 hashes of blacklisted certificates that can be used when monitoring
+  SSL/TLS certificate exchanges.
+
+inputs:
+  refresh-interval:
+    name: Time between context updates
+    description: |
+      The duration between updates that fetch the SSLBL database via the API.
+    default: 1 hour
+
+contexts:
+  feodo:
+    type: lookup-table
+    description: |
+      A table that is keyed by SHA1 hashes of SSL certificates on the SSL
+      blocklist.
+
+pipelines:
+  update-context:
+    name: Update SSLBL Context
+    description: |
+      A pipeline that periodically refreshes the SSLBL lookup table.
+    definition: |
+      // tql2
+      every {{ inputs.refresh-interval }} {
+        load_http "from https://sslbl.abuse.ch/blacklist/sslblacklist.csv"
+        read_csv comments=true, header="timestamp,SHA1,reason"
+        legacy "context update sslbl --key=SHA1"
```

Comment:
Ideally, keep as few operators as possible in `every`. The `context update` can and should be moved out.